code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,6 +319,20 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
+	if(method==INGEST && is_species(M, /datum/species/squid))
+		to_chat(M, "<span class='danger'>Your tongue shrivels as you taste the salt! It burns!</span>")
+		if(prob(25))
+			user.emote("scream")
```

Comment:
`M` not `user`. You gotta stop pulling vars out of thin air.

Reply:
fak